### PR TITLE
Purrbation works on nonhumans

### PR DESCRIPTION
**Pull request by SomeguyManperson**
# Document the changes in your pull request

die cringe

# Changelog

:cl:  
bugfix: purrbation works on nonhumans
/:cl:

### CODE REVIEW

**Comment by Chubbygummibear** [comment]
NO PLEASE I BEG YOU

**Comment by ghost** [comment]
Nooooo

File: code/modules/mob/living/carbon/human/species_types/felinid.dm [hunk 1]
```diff
@@ -83,8 +83,9 @@
 		CHECK_TICK
 
 /proc/purrbation_toggle(mob/living/carbon/human/H, silent = FALSE)
-	if(!ishumanbasic(H))
-		return
+	if(!ishumanbasic(H) && !iscatperson(H))
+		purrbation_apply_mutant(H, silent)
+		return TRUE
```
Comment (Altoids1) [inline code/modules/mob/living/carbon/human/species_types/felinid.dm:88]: My main complaint is that it looks like this doesn't really make any faithful use of the `silent` parameter, in the non-human case.

**Review by Altoids1 — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/species_types/felinid.dm (hunk 1 above)
Comment (SomeguyManperson) [reply code/modules/mob/living/carbon/human/species_types/felinid.dm:88]: addressed

**Review by ToasterBiome — APPROVED** [review]

**Comment by ghost** [comment]
Gods abandoned us

**Comment by JamieD1** [comment]
This is fucked up

**Comment by Sniblet** [comment]
This would be funny if it only affected admin purrbation but this also means that catmains have to remember to turn off purrbation any time they want to play a nonhuman race
I might genuinely be the only person affected by this but it's gonna trip me up a lot and for that reason even I hate it, can we get a nonhuman purrbation toggle

**Pull request merged**